instrumentation/aws-sdk/aws-sdk-2.2/library-instrumentor/build.gradle.kts

```diff
@@ -0,0 +1,23 @@
+plugins {
```

Comment:
Can we try importing this module in the javaagent module and deleting the duplicate interceptor? Don't think we can do it for aws sdk v1 since there is some custom logic there but guess we may as well for v2 if possible.

Reply:
It took me some time to work around the java-agent package shader but it works now.